admin/views.py

```diff
@@ -553,11 +577,19 @@ def resource_edit(resource_type, resource_id):
 
     for attribute in editable_attributes:
         attribute_value = request.form.get(attribute["name"])
-        print('attribute_value....', attribute_value)
-        validated_attribute_value = validate_resource_attribute(resource_type, attribute, attribute_value)
-        print('validated_attribute_value....', validated_attribute_value)
+        # print('attribute_value....', attribute_value)
+        validated_attribute_value = validate_resource_attribute(
+            resource_type, attribute, attribute_value
+        )
+        # print('validated_attribute_value....', validated_attribute_value)
         setattr(resource, attribute["name"], validated_attribute_value)
 
+    if attribute["name"] == "password":
```

Comment:
this is incorrect as the `attribute` is a loop variable (defined from attribute in attributes) but you're using it outside the loop. Which means the attribute must be taking the last element value when the loop ends. Which is not right because this code will break if the "password" field is not the last one in the attributes array.